src/Microsoft.DotNet.Darc/src/Darc/Options/SubscriptionStatusesCommandLineOptions.cs

```diff
@@ -7,21 +7,24 @@
 
 namespace Microsoft.DotNet.Darc.Options
 {
-    [Verb("subscription-status", HelpText = "Enables or disables a subscription matching the id.")]
-    internal class SubscriptionStatusCommandLineOptions : CommandLineOptions
+    [Verb("subscription-status", HelpText = "Enables or disables a subscriptions matching a specified criteria.")]
+    internal class SubscriptionStatusesCommandLineOptions : SubscriptionsCommandLineOptions
     {
-        [Option("id", Required = true, HelpText = "Subscription's id.")]
+        [Option("id", HelpText = "Specific subscription's id.")]
         public string Id { get; set; }
 
-        [Option('e', "enable", HelpText = "Enable subscription.")]
+        [Option('e', "enable", HelpText = "Enable subscription(s).")]
         public bool Enable { get; set; }
 
-        [Option('d', "disable", HelpText = "Disable subscription.")]
+        [Option('d', "disable", HelpText = "Disable subscription(s).")]
         public bool Disable { get; set; }
 
+        [Option('q', "quiet", HelpText = "Do not confirm which subscriptions are about to be enabled/disabled.")]
+        public bool NoConfirmation { get; set; }
+
         public override Operation GetOperation()
         {
-            return new SubscriptionStatusOperation(this);
+            return new SubscriptionStatusesOperation(this);
```

Comment:
>SubscriptionStatusesOperation [](start = 23, length = 29)

Would "SubscriptionsStatus" represent better the operations since the amount of statuses remains the same (enabled/disabled) but now more than 1 subscriptions can be edited?